gc/base/standard/Scavenger.cpp

```diff
@@ -1792,12 +1800,12 @@ MM_Scavenger::copy(MM_EnvironmentStandard *env, MM_ForwardedHeader* forwardedHea
 		} /* CS build flag  enabled: end of nested if-forwarding-succeeded check */
 #endif
 	} else { /* CS build flag  enabled: mid point of outter   if-forwarding-succeeded check
-	          * CS build flag disabled: mid point of the only if-forwarding-succeeded check */
+			* CS build flag disabled: mid point of the only if-forwarding-succeeded check */
 
 		forwardingFailed(env, forwardedHeader, destinationObjectPtr, copyCache);
 
 	} /* CS build flag  enabled: end of outter   if-forwarding-succeeded check
-	   * CS build flag disabled: end of the only if-forwarding-succeeded check */
+	* CS build flag disabled: end of the only if-forwarding-succeeded check */
```

Comment:
most if not all alignments in the comments that changed, looked to me ok already (asterisk should be vertically aligned)

Reply:
Yeah I noticed after resolving that the alignments are incorrect. Currently patching those up